core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -213,55 +237,75 @@ class UserAuthJsonSupport extends JsonSupport {
 
     override def read(json: JsValue): ProjectRole = json match {
       case JsString(name) => ProjectRoles.withName(name)
-      case _              => throw new DeserializationException("could not deserialize project role")
+      case _              => throw DeserializationException("could not deserialize project role")
     }
   }
 
-  implicit val servicePerimeterNameFormat = ValueObjectFormat(ServicePerimeterName)
+  implicit val servicePerimeterNameFormat: ValueObjectFormat[ServicePerimeterName] = ValueObjectFormat(
```

Comment:
This file also had a bunch of type annotations for public values added, contributing to the diff size.